test/mocks/SimpleERC20Wrapper.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.21;
+
+import {ERC20} from "src/token/ERC20.sol";
+import {IERC20Wrapper, IERC20Metadata} from "src/interfaces/IERC20.sol";
+import "forge-std/Test.sol";
+import {console} from "forge-std/console.sol";
```

Comment:
This is not needed. Also AFAIK you don't have to import this explicitly. It's part of the `Test.sol` anyways.